gameboard/setupships.scss

```diff
@@ -0,0 +1,221 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
+
+$base-font-color: #fff;
+$btn-color: #243c60;
+$bgc-color: rgba(0, 0, 0, 0.45);
+$btn-color-hover: #051e42;
+$base-font-family: "Lato", sans-serif;
+$bg-url: url("bg.png");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  color: $base-font-color;
+  font-family: $base-font-family;
+}
+
+.wrapper {
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  background-image: $bg-url;
+
+  &__main {
+    background-color: $bgc-color;
+    height: 100%;
+    width: 90%;
+    margin: 0 auto;
+    display: flex;
+    justify-content: center;
+    position: relative;
+  }
+
+  &__header {
+    position: absolute;
+    top: 10%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+}
+.gameBoard {
+  &__wrapper {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    // flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+  }
+
+  &__board {
+    width: 400px;
+    height: 400px;
+    background-color: #fff;
+    display: flex;
+    flex-wrap: wrap;
+    position: relative;
+    margin: 0 50px 0 0;
+
+    &-fieldsWrapper {
+      height: 100%;
+      width: 100%;
+      display: flex;
+      flex-wrap: wrap;
+    }
+
+    &-field {
+      height: 10%;
+      width: 10%;
+      border: 1px solid black;
+      &:nth-child(-n + 10) {
+        border-top: none;
+      }
+
+      &:nth-child(10n) {
+        border-right: none;
+      }
+
+      &:nth-child(10n + 1) {
+        border-left: none;
+      }
+
+      &:nth-child(n + 91) {
+        border-bottom: none;
+      }
+    }
+  }
+  &__RandomizeButton {
+    height: 20%;
+    width: 100%;
+    background-color: $btn-color;
+    border: none;
+    border-radius: 15px;
+    font-size: 48px;
+    font-weight: 700;
+    align-self: flex-end;
+    transform: translateY(40px);
+
+    &:hover {
+      cursor: pointer;
+      background-color: $btn-color-hover;
+    }
+  }
+  &__charactersWrapper {
+    position: absolute;
+    width: 40px;
+    height: 100%;
+    left: -40px;
+    display: flex;
+    flex-direction: column;
+
+    &-char {
+      height: 40px;
+      width: 40px;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+
+      h1 {
+        font-size: 24px;
```

Comment:
Move to a variable. Better - add class for all `h1` and style it with `24px` font-size ... from a variable. You do the same thing for all `h1`. This code is redundant.